Zend/zend_closures.c

```diff
@@ -467,6 +467,11 @@ ZEND_API zend_function *zend_get_closure_invoke_method(zend_object *object) /* {
 		ZEND_ACC_RETURN_REFERENCE | ZEND_ACC_VARIADIC | ZEND_ACC_HAS_RETURN_TYPE;
 
 	invoke->common = closure->func.common;
+
+	if (invoke->common.attributes) {
+		GC_TRY_ADDREF(invoke->common.attributes);
```

Comment:
Actually, I don't even know if we should be copying the attributes at all instead of `invoke->common.attributes = NULL;`... I'll have to think tomorrow

Reply:
Oh it is, via reflection, added that to the test now.

Reply:
> Oh it is, observable via reflection, added that to the test now.

Indeed, that appears to be another regression in 8.4.4: https://3v4l.org/lHnPX